website/docs/faq.md

```diff
@@ -99,6 +164,38 @@ Zombie (Stale) flags are feature flags that are not changed in the last (configu
 ### What is the Zombie (Stale) Flags Report?
 The [Zombie (Stale) Flags Report](https://app.configcat.com/my-account/zombie-flags-report) is a list of all feature flags that are not changed in the last (configurable) number of days. You can use this report to identify and remove stale feature flags from your source code. This report is weekly emailed to you. You can set your [email preferences here](https://app.configcat.com/my-account/zombie-flags-report).
 
+### I am not getting the Zombie Feature Flag email report. What am I doing wrong?
+You can change the frequency, criteria and scope of the ZFFR from the dashboard. 
+
+**Note:** Please be aware that feature flags are only treated as zombie flags if they haven't been modified (with save & publish) in the past configured timeframe. It currently doesn't have any connection with your real usage in your code. 
+
 ### How to avoid technical debt caused by feature flags?
 The [ConfigCat CLI](/advanced/code-references/overview) can scan your code, upload code references to the [ConfigCat Dashboard](https://app.configcat.com/) and notify you about stale feature flags.
 ![Code references screenshot](/assets/cli/code-refs.png)
+
+### Is there a way to compare flag statuses between two or more environments?
+Yes, you can see the state of all your Feature Flags across all your environments in our simplified [overview](https://app.configcat.com/overview).
+
+## Joining an Organization
+### Is there an expiration date for sent invitations?
+Invitations are valid for 14 days by default, but you can re-send them anytime to extend their expiration.
+
+### I can't see the organization that I just joined
+If you created a new account before joining an organization via an invite (like that's send by your company via email), then ConfigCat automatically created a free organization for you.
+
+You can still join other organizations, but in case you're not using the free organization that comes with your new account, it's best that you delete the free org to eliminate confusion.
```

Comment:
I don't get this sentence, could be simpler:
Once you find the proper organization where you are supposed to be, we recommend deleting the duplicate. Link: https://app.configcat.com/organization/preferences
But please make sure, you aren't deleting the proper one.